arviz/plots/kdeplot.py

```diff
@@ -331,12 +331,43 @@ def _fast_kde(x, cumulative=False, bw=4.5, xmin=None, xmax=None):
     return density, xmin, xmax
 
 
-@conditional_jit
+@conditional_jit(cache=True)
 def _histogram(x, n_bins, range_hist=None):
     grid, _ = np.histogram(x, bins=n_bins, range=range_hist)
     return grid
 
 
+def _cov_1d(x):
+    x = x - x.mean(axis=0)
+    fact = x.shape[0] - 1
+    by_hand = np.dot(x.T, x.conj()) / fact
+    return np.array(by_hand)
+
+
+def _cov(data):
+    if data.ndim == 1:
+        return _cov_1d(data)
+    elif data.ndim == 2:
+        x = data.astype(float)
+        avg, _ = np.average(x, axis=1, weights=None, returned=True)
+        fact = x.shape[1] - 1
+        if fact <= 0:
+            warnings.warn("Degrees of freedom <= 0 for slice", RuntimeWarning, stacklevel=2)
+            fact = 0.0
+        x -= avg[:, None]
+        x_t = x.T
```

Comment:
i'd not define `x_t` separately, since you only use it once, and `x.T` is syntactically clear.